test/unittests/state_tx_test.cpp

```diff
@@ -0,0 +1,46 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+#include <test/state/errors.hpp>
+#include <test/state/state.hpp>
+
+using namespace evmone::state;
+
+TEST(state_tx, validate_nonce)
+{
+    using evmc::operator""_address;
+
+    const BlockInfo bi{.gas_limit = 0x989680,
+        .coinbase = 0x01_address,
+        .prev_randao = {},
```

Comment:
```suggestion
```
I think you can skip default initialized fields.

Reply:
No I cannot. CI complains during build.